Ghidra/Features/Decompiler/ghidra_scripts/classrecovery/RTTIGccClassRecoverer.java

```diff
@@ -4549,9 +4559,10 @@ private Structure createSimpleClassStructure(RecoveredClass recoveredClass,
 				// the
 				// parent doesn't exist
 				if (baseClassStructure == null) {
-					throw new Exception(
+					Msg.error(this,
 						parent.getClassNamespace().getName(true) +
 							" : structure should exist but doesn't.");
+					continue;
```

Comment:
Thank you for the PR submission. I'm in the middle of refactoring some of the code in this PR so it would be easier for me to just add the suggested changes to my branch instead of trying to rebase on this review. Also, in the cases where there is no expected parent info I am removing the class from the list to process since it doesn't have expected information that is needed later. Appreciate the help!

Reply:
Please notify me in this PR when new changes will be pushed to upstream. Thank you.

Reply:
Will do!

Reply:
The changes were pushed.